src/Pages/saemsolyoo/Main/Feeds/Feed/Comments/Comments.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from "react";
+import Comment from "./Comment/Comment";
+import "./Comments.scss";
+
+class Comments extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      commentList: [],
+      commentValue: "",
+      disabled: true,
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data_saemsolyoo/commentData.json")
```

Comment:
```suggestion
    fetch("/data_saemsolyoo/commentData.json")
```
자기 자신의 public에서 가져오고 있는 내용이라면 위와 같이 생략해줄 수 있습니다.

Reply:
오! 몰랐던내용이에요! 감사합니다 ㅎㅎㅎ